pkg/views/workspace/selection/view.go

```diff
@@ -161,5 +186,31 @@ func (d ItemDelegate[T]) Spacing() int {
 }
 
 func (d ItemDelegate[T]) Update(msg tea.Msg, m *list.Model) tea.Cmd {
+	i, ok := m.SelectedItem().(item[T])
+	if !ok {
+		return nil
+	}
+
+	m.StatusMessageLifetime = time.Millisecond * 2000
+
+	var title string
+	switch msg := msg.(type) {
+	case tea.KeyMsg:
+		switch keypress := msg.String(); keypress {
+		case "x":
+			if i.markForDeletion {
+				i.title = strings.TrimPrefix(i.title, statusMessageDangerStyle("Delete: "))
+				i.markForDeletion = false
+				m.SetItem(m.Index(), i)
+				return m.NewStatusMessage(statusMessageGreenStyle("Removed workspace from Deletion List : ") + statusMessageGreenStyle(i.title))
```

Comment:
"deletion list" instead of "Deletion List". I would also remove the space before the colon.

Reply:
Fixed